src/main/java/juja/sqlcmd/Application.java

```diff
@@ -99,7 +99,7 @@ private String tableNameFormatted(String tableName) {
     }
 
     private void printTable(String tableName) {
-        String sqlQuery = "SELECT * FROM " + tableNameFormatted(tableName);
+        String sqlQuery = String.format("SELECT * FROM %1$s order by %1$s.id", tableNameFormatted(tableName));
```

Comment:
We have one parameter in String.format(). Why you use this pattern?

Reply:
It uses same value twice, I remember, it didn’t compile yesterday with one parameter. It need to be checked

Reply:
It uses same value twice, I remember, it didn’t compile yesterday with one parameter. It need to be checked

Reply:
There are two ways to user one parameter:
1) String.format("%s %<s", "hello")
2) String.format("%1$s %1$s", hello);